services/hanke-service/src/main/resources/db/changelog/changesets/078-create-paatos-table.sql

```diff
@@ -0,0 +1,38 @@
+--liquibase formatted sql
+--changeset Topias Heinonen:078-create-paatos-table
+--comment: Create a table for paatos, a single decision document that might be current or replaced.
+
+CREATE TABLE paatos
+(
+
+    id             uuid                     NOT NULL PRIMARY KEY,
+    application_id bigint                   NOT NULL,
+    hakemustunnus  varchar                  NOT NULL,
+    tyyppi         varchar                  NOT NULL,
+    tila           varchar                  NOT NULL,
+    nimi           varchar                  NOT NULL,
+    alkupaiva      date                     NOT NULL,
+    loppupaiva     date                     NOT NULL,
+    blob_location  varchar                  NOT NULL,
+    size           integer                  NOT NULL,
+    created_at     timestamp with time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
There is no `createdAt` in the entity class - should there be?

Reply:
It can be added later if it's needed for some reason.